package.json

```diff
@@ -34,13 +34,14 @@
     "react": "16.6.3",
     "recompose": "0.26.0",
     "rimraf": "2.6.2",
-    "rxjs": "5.5.10",
+    "rxjs": "6.3.3",
```

Comment:
Is this intentional? peerDep is 6.6.3

Reply:
Good catch - the peerDep should also be 6.3.3